app/views/bundlesLanding.scala.html

```diff
@@ -0,0 +1,11 @@
+@styles = {
```

Comment:
This looks remarkably similar to index.scala.html.  If all of the pages are going to be client-side rendered, can we refactor to avoid having a corresponding twirl template for each?